models/projeto_subsidio_sppo_encontro_contas/staging/rdo_correcao_rioonibus_servico_quinzena.sql

```diff
@@ -0,0 +1,30 @@
+/*
+  Etapa de tratamento com base na resposta aos ofícios MTR-OFI-2024/03024, MTR-OFI-2024/03025, MTR-OFI-2024/03026 e MTR-OFI-2024/03027 (Processo MTR-PRO-2024/06270)
+*/
+
+{% if var("encontro_contas_modo") == "_pos_gt" %}
+{{ config(alias=this.name ~ var('encontro_contas_modo')) }}
```

Comment:
Remover isso, não tem pós gt

Reply:
Isso é só por questão de execução da DAG (executar apenas quando for no modo `_pos_gt`). Se for em outro modo, não executa.